src/javascripts/jquery.tipsy.js

```diff
@@ -74,35 +74,52 @@
                     height: this.$element[0].getBoundingClientRect().height
                 });
                 
-                var actualWidth = $tip[0].offsetWidth,
-                    actualHeight = $tip[0].offsetHeight,
-                    gravity = maybeCall(this.options.gravity, this.$element[0]);
-                
-                var tp;
+                var tipCss = {};
+                var actualWidth = $tip[0].offsetWidth;
+                var gravity = maybeCall(this.options.gravity, this.$element[0]);
+
+                if (gravity.length === 2) {
+                    if (gravity.charAt(1) === 'w') {
+                        tipCss.left = pos.left + pos.width / 2 - 15;
+                    } else {
+                        tipCss.left = pos.left + pos.width / 2 - actualWidth + 15;
+                    }
+                }
+
+                var actualHeight = $tip[0].offsetHeight;
```

Comment:
Good catch, I can't see any reason for this to be separate from `actualWidth`. I'll update it.